projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.config.ts

```diff
@@ -390,6 +390,9 @@ export const tabbingOrderConfig: TabbingOrderConfig = {
   notificationPreference: [
     { value: 'Email', type: TabbingOrderTypes.CHECKBOX_WITH_LABEL },
   ],
+  notificationPreferenceV2: [
```

Comment:
[MINOR]
For consistency, let's prefix all code related to my-account-v2 with the prefix `my-account-v2` (in appropriate casing), but let's not add a `V2` prefix just to anything, because it might be confusing.
So I suggest to rename it to `myAccountV2NotificationPreference`.

Note: analogical suggestion applies to also other places in e2e tests in this PR.
Note 2:  It's not a part of our public API, so it's a [MINOR] comment.

Reply:
renamed